### PR TITLE
convert holoposter to TGUI

### PR DESCRIPTION
**Pull request by intercepti0n**
## About The Pull Request

I've converted holoposter selection screen to TGUI plus centered item icon in vending machine UI.

## Why It's Good For The Game

Holoposter selection now has an UI with an amazing preview picture.

## Testing

dreamseeker_DfcZEr5bPv

## Changelog
:cl:
refactor: converted holoposter to TGUI.
/:cl:

### CODE REVIEW

File: tgui/packages/tgui/interfaces/Holoposter.tsx [hunk 1]
```diff
@@ -0,0 +1,69 @@
+import { capitalize } from '../../common/string';
+import { useBackend } from '../backend';
+import { Box, Button, Section, Stack } from '../components';
```
Comment (intercepti0n) [inline tgui/packages/tgui/interfaces/Holoposter.tsx:3]:
```suggestion
import { Button, Section, Stack } from '../components';
```

**Comment by MLGTASTICa** [comment]
nice work

**Comment by itchybones** [comment]
Welp, you beat me to it. 👍

**Review by hyperioo — APPROVED** [review]
If you want to improve holoposter a bit more, you could replace the Process() by a proc listening to a signal that fires once per minute, I think we have one. That would avoid Process calls every tick for no real use (especially if icon is forced).

**Pull request merged**